package.json

```diff
@@ -84,5 +84,8 @@
     "*.{ts,tsx}": "eslint --cache --fix --max-warnings 0",
     "*.{css,scss,ts,tsx}": "prettier --cache --write --list-different"
   },
-  "packageManager": "yarn@4.2.2"
+  "packageManager": "yarn@4.2.2",
+  "resolutions": {
+    "@openmrs/esm-react-utils": "portal:/home/cbho/workspace/openmrs-esm-core/packages/framework/esm-react-utils"
```

Comment:
hardcoded to your workspace?

Reply:
Yeah... I can't have a correct version of this until the corresponding core PR is merged (hence it's still in draft mode). I'll change after that other PR is merged.